### ISSUE TITLE
Streams Summit

### ISSUE THREAD

**Issue by mcollina** [nodejs/readable-stream#203]
How about we do a Streams Summit to improve/document streams? It would also help onboarding new collaborators into the wg.

So it came out the idea in https://github.com/nodejs/node/issues/6178.

Is this something it might interest us to do?

**Comment by mikeal** [comment]
We also have a repo for organizing Collaborator Summits, https://github.com/nodejs/summit, and have some budget to help put them together alongside the Node.js Interactive events.

**Comment by jasnell** [comment]
@mikeal ... the VM summit folks were also talking about having a part 2. Perhaps it would make sense to simply make this a normal part of the Collaborator summit? Dedicate some time for each of the subsystem teams or working groups to meet up?

**Comment by mikeal** [comment]
@jasnell great idea! :)

**Comment by mcollina** [comment]
I agree with @jasnell. However I'm not sure that a couple of hours will be enough for digging deeper in the streams internal.

How about we try to sketch:

1. what are the outcomes of this meeting
2. who wants to attend, and where they are based
3. what are the prerequisites for this to happen, e.g. documentation, talks, slides, and who prepares those

and then we can plan it better (it will possibly be an iterative process).

I would like every member of @nodejs/collaborators that has an interest in streams but he fears touching them to join this discussion and possibly the summit.

**Comment by TheAlphaNerd** [comment]
@mikeal / @jasnell is there a plan to have the upcoming collaborator summits coincide with interactive?

**Comment by mikeal** [comment]
@TheAlphaNerd yes :)

**Comment by benjamingr** [comment]
Is this summit in-person or online?

**Comment by mcollina** [comment]
The intention is to run this in-person.

**Comment by TheAlphaNerd** [comment]
there is no reason the summit could not be streamed though. A number of people were streaming the VM summit and having their questions proxied

**Comment by mikeal** [comment]
I don't know how successful the VM Summit streaming was, I was there for a bit and actually had no idea anyone was participating that way.

The whole point of these things is to work through stuff in person, we spend literally the rest of the year working through stuff online so it seems silly to go through so much effort to stream the very few things we do in-person.

**Comment by Qard** [comment]
I watched the VM summit stream. It kind of worked, but definitely was hard to engage remotely.

**Comment by mhdawson** [comment]
I think its good to try and broadcast but at the same time we need to set the expectation that it will be hard to follow/interact.  For full interaction you'll really need to be there in person.

**Comment by mcollina** [comment]
After what we discussed on #252, we would really like to plan one.

I might throw some possibilities around conferences for which their dates for 2017 are public:

- webrebels - 1st and 2nd of June in Oslo
- jsday - 10-11th of May - Verona
- jsconfeu - 6-7th of May - Berlin

**Comment by MylesBorins** [comment]
+1 for jsconfeu

**Comment by yoshuawuyts** [comment]
Can do any of those dates. Node foundation should be able to cover expenses
for folks that need to fly over right?

**Comment by mcollina** [comment]
Some of us will meet at the collaborator summit on the 4-5 of May in Berlin.
